Classes/Recorders/PlaysLTCRecorder.cs

```diff
@@ -123,5 +123,9 @@ public class PlaysLTCRecorder : BaseRecorder {
 
         public override void GainedFocus() {
         }
+
+        public override bool? TrySaveReplayBufferAndBookmarks() {
+            
```

Comment:
Not sure here. Does PlaysLTC have such an feature? Are we still supporting it?

Reply:
What do you think @lulzsun? Time to remove it :)

Reply:
We are technically no longer supporting PlaysLTC, so we could see it being removed in the future. But let's just pretend to support this for now and deal with it later.